client/src/scss/general.scss

```diff
@@ -1,7 +1,19 @@
+body {
+  background: floralwhite;
+}
+
+header {
+  background: #aed0e8;
```

Comment:
put 

header {
  background: #aed0e8;
}
this in the `client/src/app/components/header/header.component.css` 

colors are none of my business but white looked cleaner/better on header in my opinion